lib/build.gradle

```diff
@@ -25,6 +25,10 @@ android {
             events 'passed', 'skipped', 'failed', 'standardOut', 'standardError'
         }
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
I don't understand why, this is not linked to the running environment. In general I prefer to disable that option because it can hide an unmocked object that should be mocked (when disabled an exception is thrown, so it's visible)